codec_specs.md

```diff
@@ -14,44 +14,99 @@ Each of these elements contain the same kind of data. And these data depend on t
+
+  
+**Codec ID:** V_UNCOMPRESSED  
+**Codec Name:** Video, raw uncompressed video frames  
+**Description:** The private data is void, all details about the used colour specs and bit depth are to be put/read from the KaxCodecColourSpace elements.
+
+  
+**Codec ID:** V_MPEG4/ISO/???  
+**Codec Name:** MPEG4 ISO Profile Video  
+**Description:** The stream complies with, and uses the CodecID for, one of the MPEG-4 profiles listed below.
+
+  
+**Codec ID:** V_MPEG4/ISO/SP  
+**Codec Name:** MPEG4 ISO simple profile (DivX4)  
+**Description:** Stream was created via improved codec API (UCI) or even transmuxed from AVI (no b-frames in Simple Profile), frame order is coding order
+
+  
+**Codec ID:** V_MPEG4/ISO/ASP  
+**Codec Name:** MPEG4 ISO advanced simple profile (DivX5, XviD, FFMPEG)  
+**Description:** Stream was created via improved codec API (UCI) or transmuxed from MP4, not simply transmuxed from AVI! Note there are differences how b-frames are handled in these native streams, when being compared to a VfW created stream, as here there are `no` dummy frames inserted, the frame order is exactly the same as the coding order, same as in MP4 streams!
+
+  
+**Codec ID:** V_MPEG4/ISO/AP  
+**Codec Name:** MPEG4 ISO advanced profile  
+**Description:** (Same as above)
+
+  
+**Codec ID:** V_MPEG4/MS/V3  
+**Codec Name:** Microsoft (TM) MPEG4 V3  
+**Description:** and derivates, means DivX3, Angelpotion, SMR, etc.; stream was created using VfW codec or transmuxed from AVI; note that V1/V2 are covered in VfW compatibility mode
+
+  
+**Codec ID:** V_MPEG1  
+**Codec Name:** MPEG 1  
+**Description:** The matroska video stream will contain a demuxed Elementary Stream (ES ), where block boundaries are still to be defined. Its RECOMMENDED to use MPEG2MKV.exe for creating those files, and to compare the results with selfmade implementations
+
+  
+**Codec ID:** V_MPEG2  
+**Codec Name:** MPEG 2  
+**Description:** The matroska video stream will contain a demuxed Elementary Stream (ES ), where block boundaries are still to be defined. Its RECOMMENDED to use MPEG2MKV.exe for creating those files, and to compare the results with selfmade implementations
+
+  
+**Codec ID:** V_REAL/????  
+**Codec Name:** Real Video(TM)   
+**Description:** The stream is one of the Real Video(TM) video streams listed below. Source for the codec names are from [Karl Lillevold on Doom9](http://forum.doom9.org/showthread.php?s=&amp;threadid=55773&amp;perpage=20&amp;pagenumber=2#post331855). The CodecPrivate element contains a "real_video_props_t" structure in Big Endian byte order as found in [librmff](https://github.com/mbunkus/mkvtoolnix/blob/master/lib/librmff/librmff.h).
```

Comment:
Checking with @robUx4, but I think lines 68-72 could be removed and then the line:
> The CodecPrivate element contains a "real_video_props_t" structure in Big Endian byte order as found in [librmff](https://github.com/mbunkus/mkvtoolnix/blob/master/lib/librmff/librmff.h).

could be appended to all the V_REAL descriptions below on lines 75, 80, 85, and 90.